unit_test/test_size/test_size_of_spdm_requester/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ SET(test_size_of_spdm_requester_LIBRARY
     spdm_transport_mctp_lib
     spdm_device_secret_lib_null
     intrinsiclib
-    platform_lib_null
+    platform_lib
```

Comment:
Because I have added the `__udivti3` in `platform_lib`. These `unit_test` add `platform_lib` to include the `__udivti3`. This is just a work around to include the `__udivti3`.